data.yaml

```diff
@@ -1,4 +1,22 @@
 alerts:
+  - identifier:
+    channel: operator
+    approved_at: 2021-09-17T11:00:00+01:00
+    starts_at: 2021-09-17T11:00:00+01:00
+    cancelled_at: 2021-09-17T11:15:00+01:00
+    finishes_at: 2021-09-17T11:15:00+01:00
```

Comment:
Optional: this would actually be the start time + the hard-coded max duration. We don't show this data anywhere, so it's not a big deal.

(Similarly with the alert below.)

Reply:
Oh thanks for pointing out! I did not realise that. I won't make the change as in time these will be replaced with correct data from the API if used at all.